puppet-squid/src/main/java/com/iadams/sonarqube/puppet/api/PuppetGrammar.java

```diff
@@ -198,15 +193,7 @@ public static void grammar(LexerfulGrammarBuilder b) {
 
         b.rule(PARAMS).is(b.optional(
                 PARAM,
-                b.zeroOrMore(COMMA, PARAM))).skip();
```

Comment:
@iwarapter Please, carefully double-check the grammar update.
`b.rule(ANY_PARAM).is(b.firstOf(PARAM, ADD_PARAM)).skip();` was useless from my point of view as it would always take PARAM and never ADD_PARAM. Like that, it simplifies the way to write rules as there's now only one "type" of parameter node.